www/content/posts/2020-5-15-kutty-0.0.1-is-released.md

```diff
@@ -7,7 +7,7 @@ tag = ["posts", "announcements"]
 
 ## Kutty 0.0.1 Release
 
-I'm pleased to announce the [0.0.1 release](https://unpkg.com/browse/kutty.org@0.0.1/) of kutty, the successor
+I'm pleased to announce the [0.0.1 release](https://cdn.jsdelivr.net/npm/browse/kutty.org@0.0.1/) of kutty, the successor
```

Comment:
This is erroring: `Couldn't find the requested file /kutty.org@0.0.1/ in browse.`

Actually, even unpkg is not finding it now. I suggest that we just don't touch the old release notes.

Reply:
OK, let me drop those changes in the changelog.

Reply:
@yawaramin The PR has been rebased and updated.